client/src/domain/event/EventLocation.tsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import Map from "../../common/components/map/Map";
+import { EventDetailsQuery } from "../../generated/graphql";
+import AngleRightIcon from "../../icons/AngleRightIcon";
+import ExternalLinkIcon from "../../icons/ExternalLinkIcon";
+import LocationIcon from "../../icons/LocationIcon";
+import getLocale from "../../util/getLocale";
+import getLocalisedString from "../../util/getLocalisedString";
+import styles from "./eventLocation.module.scss";
+
+interface Props {
+  eventData: EventDetailsQuery;
+}
+
+const EventLocation: React.FC<Props> = ({ eventData }) => {
+  const { t } = useTranslation();
+  const locale = getLocale();
+
+  const getGoogleLink = () => {
```

Comment:
Could these `getX`-functions be declared outside this function? Now the `getX`-functions are recreated on every render.

Reply:
Good point. Already fixed that on PR13